app/tiebout.hoon

```diff
@@ -0,0 +1,253 @@
+::
+::::  /app/tiebout/hoon
+::
+/?  309
+/-  hall
+/+  tiebout
+=,  tiebout
+=,  eyre
+::
+::  state:
+::
+~%  %tiebout  ..^is  ~
+|_  [bol=bowl:gall sta=state]
```

Comment:
I disagree with this, having a face for state means makes things clearer. I've heard other people argue both sides of this before. Perhaps this is something that should be resolved by a style guide. As someone who has had to update someone else's =* lark expression, I'm against that solution.

Reply:
Agreed that `=*`ing some lark is bad. I've felt your pain before. (But never in this context, because the lark here practically never changes.)

ime this is a non-issue for top-level app cores though. Have never ever needed a face on app state. Clarity is resolved by "common pattern" and having the state structure in the same file.

Surprised to hear others have argued for adding a face in this context, would be curious to hear from them.

Reply:
Keeping the face.